lib/header.js

```diff
@@ -40,7 +40,7 @@ function addLinks (res, fileMetadata) {
     addLink(res, 'http://www.w3.org/ns/ldp#DirectContainer', 'type')
   }
   if (fileMetadata.isStorage) {
-    addLink(res, 'http://www.w3.org/ns/pim/space#Storage', 'type')
+    addLink(res, 'http://www.w3.org/ns/pim/space#storage', 'type')
```

Comment:
Why lowercase? http://www.w3.org/ns/pim/space#storage is a property while http://www.w3.org/ns/pim/space#Storage is the class.

this is what CSS sends in the header:

```
link: <http://www.w3.org/ns/pim/space#Storage>; rel="type"
```

Reply:
Because NSS in profile/card uses space:storage, ESS also and the ontology also.
I think it is an error in CSS.

Reply:
1. The header on </>
```
   link:  <http://www.w3.org/ns/pim/space#Storage>; rel="type".
````
Is the same as the predicate in the turtle of </.meta>
```
   </> rdfs:type  <http://www.w3.org/ns/pim/space#Storage> .
```


2. OTOH, in the profile, the owner/server MAY have a  triple like.
```
   <#me> pim:storage </>.
```
#1 asserts that we are at the Pod root.  #2 asserts that WebId owns the pod.  Some owners may not want to have that fact advertised and there is no requirement that they do.

Reply:
There is a convention to have predicates begin with a lower-case letter, and classes begin with an uppercase letter, and that's what we're seeing here: `space:Storage` is a class, `space:storage` is a predicate, and both are in use. And indeed, as @jeff-zucker is saying, the `Link` header are using classes, so it should be upper case, like @angelo-v objected.

Reply:
Confirming that the Solid Protocol requires `http://www.w3.org/ns/pim/space#Storage` (uppercase 'S') for Link rel=type.

```suggestion
    addLink(res, 'http://www.w3.org/ns/pim/space#Storage', 'type')
```